stdlib/array.gr

```diff
@@ -681,6 +752,9 @@ provide let count = (fn, array) => {
  * @param fn: The function to call on each element, where the returned value indicates if the element satisfies the condition
  * @param array: The array to iterate
  * @returns The total number of elements that satisfy the condition
+ * 
+ * @example
+ * Array.counti((a, i) => a >= 7 && i > 2, [> 5, 6, 7, 8, 9]) // 2
```

Comment:
Same As Above:
```suggestion
 * Array.counti((a, i) => a >= 7 && i > 2, [> 5, 6, 7, 8, 9]) == 2
```

Reply:
Same comment as before. This is not returning void so simply in line the result.....